lib/launcher.js

```diff
@@ -218,7 +218,9 @@ class Launcher {
 
     async writeThemeFiles () {
         info('Updating theme files')
-        const sourceDir = path.join(__dirname, '..', 'node_modules', '@flowforge', 'nr-theme')
+        const sourceDir1 = path.join(__dirname, '..', 'node_modules', '@flowforge', 'nr-theme')
+        const sourceDir2 = path.join(__dirname, '..', 'node_modules', '@flowforge', 'nr-theme')
```

Comment:
```suggestion
        const sourceDir1 = path.join(__dirname, '..', 'node_modules', '@flowforge', 'nr-theme')
        const sourceDir2 = path.join(__dirname, '..', '..', 'nr-theme')
```

Is this a copy paste error Ben?